src/controllers/user.ts

```diff
@@ -0,0 +1,90 @@
+import {checkoutUserData, deleteUserData, getUserById, updateUserData} from "../services/user";
+import {Request, Response} from "express";
+import {returnUnauthorized} from "../utils/return-unauthorized";
+import {hideDeleteStatus} from "../utils/hide-delete-status";
+
+export const ID_HEADER_NAME = "x-user-id"
```

Comment:
If you use a constant in different files, then it would be better to store it in the separate file named `constants.ts` in `src` folder.